modules/flowable-cmmn-engine/src/main/resources/org/flowable/cmmn/db/mapping/common.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+
+<mapper namespace="org.flowable.cmmn.db.mapping">
+    
+    <sql id="changeTenantIdCaseCriteriaSql">
```

Comment:
The pattern used here is not used in other places of the Flowable codebase. The pattern being, having a common sql in one file which is then reused from other MyBatis mappings. We do have one common.xml, which is for the native queries, but that one is simpler and doesn't use any column names or table names.

Let's move the logic to the respective MyBatis configurations for each entity.

Reply:
Done